src/hooks/use-link-delayed.ts

```diff
@@ -53,6 +53,15 @@ export const useLinkDelayed = ({
                 return;
             }
 
+            // If the link is an external link, open in new tab.
+            if(goTo.includes("://")) {
+                if (window !== undefined) {
```

Comment:
This might cause a runtime error. Ideally you could use typeof check

```suggestion
                if (typeof window !== "undefined") {
```